### PR TITLE
FoldableLayout Code Refactory, ktlint fixes, tab layout to wm beta2

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

**Review by bimiron — APPROVED** [review]
Except for the sample packages which we discussed to change in another PR, everything looks ok to me:)

**Review by soalb-m — APPROVED** [review]

File: layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutController.kt [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+            width = MATCH_PARENT
+            height = MATCH_PARENT
+        }
+    }
+}
+
+/**
+ *  Clears the content, inflate the new content and add it to the view.
+ */
+private fun ViewGroup.addContent(@LayoutRes layoutResId: Int): View? {
```
Comment (CesarValiente) [inline layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutController.kt:333]: I think indentation starts getting odd here.

**Review by CesarValiente — APPROVED** [review]
LGTM! Jus tone thing about indentation in one class. Please Andrei, check before you merge it.

Weird that ktlint hasn't complain about that. I will check the rules.

File: layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutController.kt (hunk 1 above)
Comment (ancirja-m) [reply layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayoutController.kt:333]: This is an extension function, so it's outside of the class

**Pull request merged**